doc/build/build.md

```diff
@@ -91,69 +91,47 @@ From the root directory of the repository:
 ```bash
 # With docker daemon running
 multi-cluster-app-dispatcher % make images
-
 ....
-
 # output from a local branch, MacOS build, local file names replaced with XXXXXXXXXX
 "---"
 "MAKE GLOBAL VARIABLES:"
 "  "BIN_DIR="_output/bin"
-"  "GIT_BRANCH="local_e2e_test"
+"  "GIT_BRANCH="issue_315_small_changes"
```

Comment:
It's an example of branch, I am not sure what is your concern. Can you provide more insight ?

Reply:
Well, the name of the `GIT_BRANCH` has `issue_315` in it. I imagine at some point that will be a resolved issue and thus this name won't be relevant anymore, so I'm wondering if it would be better to have a different name. I also noticed it the name `issue_315_small_changes` is used in a lot of parts so that, when new issues come are we going to have to update those names to point to the new issue branch? My concern is with the naming itself, not with the logic if that helps.

Reply:
@dmatch01 What is your take on this discussion ? I am not to sure of the value of adding the examples, but I will defer to your better judgement.

Reply:
@dmatch01 What is your take on this discussion ? I am not to sure of the value of adding the examples, but I will defer to your better judgement.

Reply:
We are nearing the release cut soon. I am ok with @z103cb suggestion, @metalcycling If needed we can certainly change it post-release, please open an issue to track the same, thanks